eve/workers/openstack-terraform/terraform/scripts/network.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -xue -o pipefail
+
+IFACE=$1
+
+MAC_ADDR="$(cat "/sys/class/net/$IFACE/address")"
```

Comment:
Useless use of cat & subshell:
```suggestion
read -r MAC_ADDR < "/sys/class/net/$IFACE/address"
```

Reply:
Right :+1: